util/session/sessionmanager.go

```diff
@@ -400,7 +401,13 @@ func (mgr *SessionManager) VerifyUsernamePassword(username string, password stri
 		return err
 	}
 	if !account.Enabled {
-		return status.Errorf(codes.Unauthenticated, accountDisabled, username)
+		log.Errorf(accountDisabled, username)
```

Comment:
We should not print error message if user is trying to login using invalid username/password. Please remove `log.Errorf`

Reply:
And here also, a `return status.Errorf(codes.Unauthenticated, invalidLoginError, username)`. In both cases (this and the above one) I think we should log the error (i.e. whether account is disabled, or login not allowed) to prevent user enumeration (i.e. you can enumerate users which have apiKey permissions this way)